scripts/CEE/get-rhoc-connector-info/metadata.yaml

```diff
@@ -0,0 +1,86 @@
+file: get-rhoc-connector-info.sh
+name: get-rhoc-connector-info
+shortDescription: Collect information for the specific conenctor instance.
+description: |
+  Collect information for the specific conenctor instance.
+  Data is stored in a directory named "inspect.${connector_id}.${timestamp}"
+author: lburgazz
+allowedGroups:
+  - SREP
+  - CSSRE
+  - CEE
```

Comment:
@pat-cremin should we be including MT-SRE as part of the allowed groups now?

Reply:
I guess that does make sense even if we are not yet fully integrated, they will likely need this in the near future. 

@lburgazzoli can you add MTSRE to the groups here.

Reply:
@pat-cremin done